### PR TITLE
Fix 'UndoRedo' increasing its version on actions that should be merged

### PR DESCRIPTION
**Pull request by YeldhamDev**

### CODE REVIEW

File: core/undo_redo.cpp [hunk 1]
```diff
@@ -396,6 +401,7 @@ UndoRedo::UndoRedo() {
 	action_level = 0;
 	current_action = -1;
 	merge_mode = MERGE_DISABLE;
+	merge_version = false;
```
Comment (reduz) [inline core/undo_redo.cpp]: I would rename to just "merging", other than that it seems fine.

**Review by reduz — DISMISSED** [review]

**Comment by YeldhamDev** [comment]
@reduz Changes made.

**Comment by akien-mga** [comment]
Thanks!

**Comment by hpvb** [comment]
Consider for 3.1.2

**Pull request merged**